agile_problem3_test.exs

```diff
@@ -0,0 +1,22 @@
+ExUnit.start
+
+defmodule Problem3Test do
+	use ExUnit.Case
+	
+	test "Problem 3: positive test" do
+		assert Problem3.check_duplicates([21, 13, -20, 4, 20, 93, 100, -19, -95, -83, 3, -3, 42, -26, 100, 42, -52, 46, -91, -23]) == 1
+	end
+	test "Problem 3: two element test" do
+		assert Problem3.check_duplicates([1,2,3,4,5,6]) == 0
+	end
+	test "Problem 3: negative test - one element" do
+		Problem3.check_duplicates([-30, -15, -122, 0, 1001]) == 0

```

Comment:
Not a valid use case, the problem states `Given an array of numbers having size N in which every number is between 1 and N`

You don't have negative numbers
If you have the number 1001 you need to have an array with at least 1001 elements.